rotkehlchen/tests/utils/dataimport.py

```diff
@@ -2514,3 +2514,244 @@ def assert_kucoin_import_results(rotki: Rotkehlchen):
         fee_currency=A_USDT,
     )]
     assert trades == expected_trades
+
+
+def assert_blockpit_import_results(rotki: Rotkehlchen):
+    """A utility function to help assert on correctness of importing data from blockpit"""
+    dbevents = DBHistoryEvents(rotki.data.db)
+    with rotki.data.db.conn.read_ctx() as cursor:
+        trades = rotki.data.db.get_trades(cursor, filter_query=TradesFilterQuery.make(), has_premium=True)  # noqa: E501
+        asset_movements = rotki.data.db.get_asset_movements(
+            cursor,
+            filter_query=AssetMovementsFilterQuery.make(),
+            has_premium=True,
+        )
+        history_events = dbevents.get_history_events(cursor, filter_query=HistoryEventFilterQuery.make(), has_premium=True)  # noqa: E501
```

Comment:
okay you seem to do it a lot. Just look at the PR and fix all places where you are not using kwargs.

```suggestion
        history_events = dbevents.get_history_events(cursor=cursor, filter_query=HistoryEventFilterQuery.make(), has_premium=True)  # noqa: E501
```